{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:sentry:sentry:*:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/getsentry/sentry/commit/fad12c1150d1135edf9666ea72ca11bc110c1083",
        "version_end_excluding": {
            "sentry:sentry": "23.7.2"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "PYTHON",
    "licences": {
        "database": "",
        "affected_repository": "Other"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2023-39349"
        ],
        "CWE": [
            "CWE-284",
            "CWE-287"
        ]
    }
}